scripts/bump_cloudbeat.sh

```diff
@@ -21,8 +21,16 @@ echo "NEXT_CLOUDBEAT_VERSION: $NEXT_CLOUDBEAT_VERSION"
 echo "CURRENT_CLOUDBEAT_VERSION: $CURRENT_CLOUDBEAT_VERSION"
 echo "CURRENT_MINOR_VERSION: $CURRENT_MINOR_VERSION"
 
+# clear branches if they exists
+branches=("$NEXT_CLOUDBEAT_BRANCH" "$NEXT_CLOUDBEAT_HERMIT_BRANCH" "$RELEASE_CLOUDBEAT_BRANCH")
+for branch in "${branches[@]}"; do
+    if git ls-remote --exit-code --heads origin "$branch"; then
+        git push origin --delete "$branch"
+    fi
+done
```

Comment:
the script is intended to be idempotent so if we run it multiple times, the result will be the same. to do so, we remove any existing branches (which will automatically close open PRs)

the thinking here was that if the script ran the first time and partially failed, for example - the user ran it before branching out so the release branch didn't exist, then after they create the release branch (or fix any other error, they can run it again.